sockeye/inference.py

```diff
@@ -1435,26 +1441,23 @@ def translate(self, trans_inputs: List[TranslatorInput]) -> List[TranslatorOutpu
                             "constraint" if len(trans_input.constraints) == 1 else "constraints",
                             ", ".join(" ".join(x) for x in trans_input.constraints))
 
+        num_bad_empty = len(translated_chunks)
```

Comment:
Is this correct? Perhaps the github diff doesn't show if this is modified somewhere else.

Reply:
it should be. translated_chunks is populated above for bad and empty inputs and then extended below for the actual translations.